firmware/application/clock_manager.cpp

```diff
@@ -463,3 +463,17 @@ void ClockManager::stop_audio_pll() {
 	cgu::pll0audio::power_down();
 	while( cgu::pll0audio::is_locked() );
 }
+
+void ClockManager::enable_clock_output(bool enable) {
+	if(enable) {
+		clock_generator.enable_output(clock_generator_output_clkout);
+		clock_generator.set_ms_frequency(clock_generator_output_clkout, 10000000, si5351_vco_f, 0);
+	} else {
+		clock_generator.disable_output(clock_generator_output_clkout);
+	}
+
+	if(enable)
```

Comment:
Is there a reason for the same if(enable) twice ?

Reply:
yeah it is duplicated, unless there is an obscure reason why